specification/registry/xr.xml

```diff
@@ -5748,6 +5748,35 @@ maintained in the default branch of the Khronos OpenXR GitHub project.
             <component                                subpath="/output/haptic"            type="XR_ACTION_TYPE_VIBRATION_OUTPUT"/>
         </interaction_profile>
 
+        <!-- XR_OPPO_controller_interaction -->
+
+        <interaction_profile name="/interaction_profiles/oppo/mr_controller_oppo" title="OPPO MR Controller">
```

Comment:
@rpavlik I have a special question about new interactioon profile: is it recommended to add new interaction profile path to HelloXR's suggestion list?

Reply:
I don't think so, the actions in hello xr don't really need more than what is in the simple controller profile so adding more just makes it  more complicated. I haven't really seen people add to that regularly.

Reply:
Thanks for clarifying it.

Reply:
Hi @rpavlik, this is Bowie (Buyi) from OPPO, I am a colleague of @hjiang36, if there is anything that needs to be changed, please let us know. Thanks.